scripts/build.sh

```diff
@@ -7,18 +7,10 @@ current=$(node -e "console.log(require('./package.json').version)")
 echo "Releasing: $current"
 echo ""
 
-# Build base image
-
-docker build \
-  --progress plain \
-  -t algolia/renderscript-pw-chromium:${current} \
-  --build-arg "VERSION=${current}" \
-  -f Dockerfile.pw \
-  .
-
 # Build renderscript
 
-docker build \
+docker buildx build \
```

Comment:
I'm getting an error when trying to build the image locally, do I need to update docker ?
`error: multiple platforms feature is currently not supported for docker driver. Please switch to a different driver (eg. "docker buildx create --use")`

Reply:
mmh good question, I think I had this warning too and just did `docker buildx create --use` solved it but I don't know what it does internally :D